### ISSUE TITLE
HELP WANTED: Force trade definition

### ISSUE THREAD

**Issue by wilburforce83** [binary-com/binary-bot#928]
Is there a way in the blockly interface to force it to run `Define Trade Options:` without a trade triggering the `Trade Again` block?

I use a strategy which changes the barrier offset set in `Define Trade Options:` based on MA postions, this runs once on start, and after each trade. If the MA is about to cross I find myself in a position where the trade is defined in a direction which won't be traded.

Basically, I want to be able to break the loop and start from `define trade options` say every 2 minutes in order to make sure the trading direction / offset is kept up to date without having to complete a trade first.

Thanks!!!

**Comment by wilburforce83** [comment]
Being an idiot, I have just created a function to change barrier under the `This block is called for every tick` to keep the trading direction and offset up to date. Closed
